.github/CONTRIBUTING.md

```diff
@@ -175,6 +175,22 @@ to verify ZFS is behaving as intended.
 
 ## Style Guides
 
+### Repository Structure
+
+OpenZFS uses a standardised branching structure.
+- The "development and main branch", is the branch all development should be based on.
+- "Release branches" contain the latest released code.
+- "Staging branches" contain selected commits prior to being released
+- Backports are special branches centered around supporting older releases by porting patches
+
+
+Development and Main branch: `master`
```

Comment:
I believe you need two spaces at the end of the line if you intended for these each to be rendered on its own line.